### ISSUE TITLE
grad_outputs for gradient penalty 

### ISSUE THREAD

**Issue by leesunfreshing** [kuc2477/pytorch-wgan-gp#1]
Hi, could you specify the purpose of grad_outputs? It is not clear given the PyTorch Tutorial... Moreover, why it should be torch.ones for penalty item?
